dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/CreateFileMapper.java

```diff
@@ -4,38 +4,36 @@
  */
 package com.linkedin.dynamometer.workloadgenerator;
 
-import com.google.common.collect.Lists;
 import java.io.IOException;
 import java.io.OutputStream;
 import java.net.URI;
 import java.util.List;
-import java.util.concurrent.TimeUnit;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.io.LongWritable;
 import org.apache.hadoop.io.NullWritable;
 import org.apache.hadoop.mapreduce.Mapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 
 /**
  * <p>CreateFileMapper continuously creates 1 byte files for the specified duration to increase the
- * number of file objects on the NN.
+ * number of file objects on the NN. This uses {@link TimedInputFormat}; see its Javadoc for configuration
+ * information.
  *
  * <p>Configuration options available:
  * <ul>
- *   <li>{@value NUM_MAPPERS_KEY} (required): Number of mappers to launch.</li>
- *   <li>{@value DURATION_MIN_KEY} (required): Number of minutes to induce workload for.</li>
  *   <li>{@value SHOULD_DELETE_KEY} (default: {@value SHOULD_DELETE_DEFAULT}): If true, delete the files
  *       after creating them. This can be useful for generating constant load without increasing the
  *       number of file objects.</li>
  *   <li>{@value FILE_PARENT_PATH_KEY} (default: {@value FILE_PARENT_PATH_DEFAULT}): The root directory
  *       in which to create files.</li>
  * </ul>
  */
-public class CreateFileMapper extends WorkloadMapper<NullWritable, NullWritable> {
+public class CreateFileMapper extends WorkloadMapper<LongWritable, NullWritable> {
 
-  public static final String NUM_MAPPERS_KEY = "createfile.num-mappers";
-  public static final String DURATION_MIN_KEY = "createfile.duration-min";
   public static final String FILE_PARENT_PATH_KEY = "createfile.file-parent-path";
   public static final String FILE_PARENT_PATH_DEFAULT = "/tmp/createFileMapper";
```

Comment:
not related to this PR, but better not hard code /tmp, even for default. Use java tmp directory as prefix.

Reply:
This is `/tmp` on the NameNode under test, not on the local file system. Using Java tmp directory does not make sense here.